scripts/eplus_dump_clustering_export.py

```
@@ -0,0 +1,354 @@
+
+        feature_rows.append(row)
+
+        # Long timeseries (numeric columns only)
+        melt_df = df.reset_index()
+        ts_col = melt_df.columns[0]
+        melt_df = melt_df.rename(columns={ts_col: "timestamp"})
+        id_vars = ["timestamp"]
+        val_cols = [c for c in melt_df.columns if c not in id_vars]
+        if val_cols:
+            chunk = melt_df.melt(id_vars=id_vars, value_vars=val_cols, var_name="col", value_name="value")
+            chunk["role"] = chunk["col"].map(lambda c: role_map.get(str(c), str(c)))
```

Ruff (0.16.1)

[warning] Function definition does not bind loop variable `role_map`

(B023)